src/test/java/uk/co/ramp/covid/simulation/output/RStatsTest.java

```diff
@@ -48,25 +48,17 @@ public void meanRPositiveWhenInfectionsOccur() {
 
     @Test
     public void testMeanRWithLockdown() {
-        int populationSize = 50000;
-        pop = PopulationGenerator.genValidPopulation(populationSize);
-        int startLock = 30;
-        int nDays = 90;
+        int startLock = 20;
+        int nDays = 40;
         pop.seedVirus(10);
         pop.getLockdownController().addComponent(
                 new FullLockdownEvent(Time.timeFromDay(startLock), pop, 2.0));
         pop.simulate(nDays);
         RStats rs = new RStats(pop);
-        double peakR = 0.0;
 
-        for (int i = 0; i < nDays; i++) {
-            //Get the peak R value before lockdown starts
-            if (i < startLock) peakR = Math.max(peakR, rs.getMeanRBefore(i + 1));
-            //Test that the R value during lockdown is always below the peak R
-            if (i > startLock + 5) {
-                assertTrue("Peak R occurred at Day " + i, rs.getMeanRBefore(i + 1) < peakR);
-            }
-        }
+        double meanRBeforeLockdown = rs.getMeanRBefore(startLock);
+        double meanRDuringLockdown = rs.getMeanRBetween(startLock, nDays);
 
+        assertTrue(meanRDuringLockdown < meanRBeforeLockdown);
```

Comment:
The intent of this test has been changed. It was designed to check that the R value during lockdown never exceeds the peak R before lockdown. However, maybe this test is better? @paulbessell would you like to comment?

Reply:
Just to note my intention on changing the test (I'm not against reverting this etc!):

We don't have a way (anymore - believe Paul removed this) to get the specific R for a day. Instead we took the mean between day 0 and some point in time (for the first 5% of infections/before lockdown). Think this means that for the old test the "peak" R was the maximum mean value (possibly overly biased to the start of infection since it's likely sensitive to outliers). Because of this, even when we were asserting that after lockdown the R goes down this *included* the pre-lockdown R values averaged out (so we were in some sense already testing that the mean falls during lockdown).
Plan was the new test was to make this idea of the "mean falling" more explicit.

For a small-ish population this seems to roughly half (which is a bit weaker than the 3 -> 1ish you might expect with more realistic data, but shows lockdowns are having an effect).

Again, happy to revert this etc after discussions!

Reply:
We might also want to check that this "mean" R doesn't affect our run output. But I think the idea was to move to secondary infections on a specific day later in the run as a more stable value (again @paulbessell  reshuffled this bit so I'm not 100% sure the intention)

Reply:
It sounds as if the original test was a bit flawed and the new test makes more sense.

Reply:
This looks generally fine and I think thta may of my concerns around the R stats were because I had not thought through the epidmeiology for the most part.

Suggested some small changes because secondary infections happen 5-20 days following infection, so R is lower befor elockdown and hence discarding these days form the stats. Likewise, no point in considering R in the 20 days before the end of the simulation because there is no infecting here.